index.js

```diff
@@ -27,6 +27,7 @@ var manifest = {
   createWriteStream: 'sink',
   del: 'async',
   get: 'async',
+  getFeedState: 'async',
```

Comment:
If I'm correct this exposes this over muxrpc. Do we want to or need to do this?
If we don't absolutely need to I would rather not, because it's harder to take back than to add later

Reply:
Have disabled

Reply:
My policy is to expose everything over MuxRPC that's exported by the JS module unless it doesn't work over the network (e.g. `_flumeUse()`). What's the benefit to maintaining two similar-but-different APIs here?